features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -24,7 +24,7 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
     Then I should see "Edit first fandom Tag"
     
     # assigning media to a fandom
-    When I fill in "Media" with "TV Shows"
+    When I fill in "tag[media_string]" with "TV Shows"
```

Comment:
So this is because in the past the tag wasn't properly given the "Uncategorized Fandoms" so now it is taking tag[media_string] as the field if you look at 

view-source:http://test.archiveofourown.org/tags/Arrow%20(TV%202012)/edit for example